packages/breadcrumb/components/Breadcrumb.tsx

```diff
@@ -9,8 +18,34 @@ export interface BreadcrumbProps {
 class Breadcrumb extends React.PureComponent<BreadcrumbProps, {}> {
   public render() {
     const { children } = this.props;
+    const breadcrumbSeparator = (
+      <Icon shape={SystemIcons.CaretRight} size={iconSizeXs} />
+    );
+    const crumbsArr = React.Children.toArray(children).reduce(
+      (acc, crumb, i) => {
+        acc.push(crumb);
+        if (i !== React.Children.toArray(children).length - 1) {
```

Comment:
`React.Children.toArray(children)` will most likely never be a bottleneck... i'd still feel better if we'd pull it out of the reduce.